chatwoot/chatwoot
# Feature/937 unmute muted contacts

# Pull Request Template

## Description
Add a button in chat's "more" menu that allows to unmute conversation(only unmute, without reopening it)
![unmute-conversation](https://user-images.githubusercontent.com/9462185/95181045-6ccc0e80-07cb-11eb-9a80-914f15823eb1.gif)

Fixes #937 

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?
- Open any chat that is not muted
- Mute it
- See that an "Unmute conversation" button appeared in the chat's "more" menu
- Click "Unmute conversation"
- See that the chat is not muted any more 

## Checklist:
